### PR TITLE
Hotfix/send correct ordinal

### PR DESCRIPTION
**Pull request by teebszet**
# 🔘 PR Type

- [x] Bugfix
- [x] Other: hotfix release

# 📜 Background

hotfix PR for: https://github.com/secretkeylabs/xverse-web-extension/pull/539

- branched off commit for tag 0.14.0
- cherry-picked commit from @victorkirov 's PR
- created and pushed new tag 0.14.1
- next: 
  - verify fix on https://github.com/secretkeylabs/xverse-web-extension/pull/539
  - verify fix on 0.14.1
  - create the Github release 0.14.1
  - submit to Chrome Web Store

# 🔄 Changes
- see https://github.com/secretkeylabs/xverse-web-extension/pull/539

Impact:
- send ordinals flow

# 🖼 Screenshot / 📹 Video
Include screenshots or a video demonstrating the changes. This is especially helpful for UI changes.

# ✅ Review checklist
Please ensure the following are true before merging:

- [x] Code Style is consistent with the project guidelines.
- [x] Code is readable and well-commented.
- [x] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [x] Code does not introduce new technical debt or issues.

### CODE REVIEW

**Comment by teebszet** [comment]
tested manually and send ordinal flow was fixed. see discord https://discord.com/channels/895889501560377364/1135491935314907217/1135563833210065006

tested with 0.14.0 here:
https://mempool.space/tx/ff1563d6a6c62abfb8b924c655b7ff56989243f4d93b8bbf5af81d1ac85e56c2#flow=&vout=0

tested with 0.14.1 here:
https://mempool.space/tx/dcb6058e6f9e455bce2c50db8c5860b64f95a39bb1c3cdda5610fa2e5f4803b6

**Review by yknl — APPROVED** [review]

**Comment by DuskaT021** [comment]
this pr has merge conflicts

File: src/app/screens/sendOrdinal/index.tsx [hunk 1]
```diff
@@ -119,23 +119,27 @@ function SendOrdinal() {
...
-        btcAddress,
-        Number(selectedAccount?.id),
-        seedPhrase,
-        network.type,
-        [ordUtxo],
+  } = useMutation<SignedBtcTx, ResponseError, string>({
+    mutationFn: async (recipient) => {
+      const addressUtxos = await btcClient.getUnspentUtxos(ordinalsAddress);
+      const ordUtxo = addressUtxos.find(
+        (utx) => `${utx.txid}:${utx.vout}` === selectedOrdinal?.output,
```
Comment (yknl) [inline src/app/screens/sendOrdinal/index.tsx:126]: Why do we name it utx instead of utxo here?

**Comment by yknl** [comment]
Can we resolve conflict and merge? The release has been submitted.

**Pull request merged**